QUICKSTART.md

```diff
@@ -0,0 +1,197 @@
+# Quick Start Guide
+
+A quick start guide to getting a system running with the latest TrenchBoot
+Secure Launch bits. Note that this is a bare bones document meant to help
+someone get up and running with Secure Launch. It does not contain detailed
+descriptions of all the technologies and terminology involved in doing a
+secure late launch. The repository this document resides in contains a plethora
```

Comment:
Why sometime we write Secure Launch and sometimes with small letters, like "secure late launch"?

Reply:
I guess I was trying to say something else but it ended up being unclear. I can just change it.